src/Fast.xs

```diff
@@ -193,8 +193,22 @@ SV *Redis__Cluster__Fast_connect(pTHX_ Redis__Cluster__Fast self) {
 }
 
 cluster_node *get_node_by_random(pTHX_ Redis__Cluster__Fast self) {
-    uint32_t slot_num = (uint32_t) (Drand01() * REDIS_CLUSTER_SLOTS);
-    return self->acc->cc->table[slot_num];
+    cluster_node *selected;
+    cluster_node *candidate;
+    int node_count;
+    nodeIterator ni;
+
+    initNodeIterator(&ni, self->acc->cc);
```

Comment:
[MEMO]

Slaves may be selected now, but previously only the master node will be selected.